drivers/input/input_touch.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/input/input_touch.h>
+
+void input_touchscreen_report_pos(const struct device *dev, uint32_t x, uint32_t y)
+{
+	const struct input_touchscreen_common_config *cfg = dev->config;
+	const uint32_t reported_x_code = cfg->swapped_x_y ? INPUT_ABS_Y : INPUT_ABS_X;
+	const uint32_t reported_y_code = cfg->swapped_x_y ? INPUT_ABS_X : INPUT_ABS_Y;
+	const uint32_t reported_x = cfg->inverted_x ? cfg->screen_width - x : x;
+	const uint32_t reported_y = cfg->inverted_y ? cfg->screen_height - y : y;
+
+	input_report_abs(dev, reported_x_code, reported_x, false, K_FOREVER);
+	input_report_abs(dev, reported_y_code, reported_y, false, K_FOREVER);
```

Comment:
Usually you want to block, but maybe there is a use case for passing in the timeout explicitly?

Reply:
Forgot to use the timeout

Reply:
`timeout` argument is now being used